dashboard/app/models/concerns/serialized_properties.rb

```diff
@@ -51,7 +51,7 @@ def define_methods_for_property(property_name)
     def define_methods_for_encrypted_property(property_name)
       cleartext_property_name = property_name.gsub(ENCRYPTED_PROPERTY_REGEX, '')
 
-      define_method(cleartext_property_name) do |*args|
+      define_method(cleartext_property_name) do |*_args|
```

Comment:
Nit: argument seems to be optional (see http://ruby-doc.org/core-2.2.0/Module.html#method-i-define_method).